addons/fluxcd/template.cue

```diff
@@ -0,0 +1,39 @@
+package main
+
+output: {
+	apiVersion: "core.oam.dev/v1beta1"
+	kind:       "Application"
+	spec: {
+		components: [
+			{
+				type: "k8s-objects"
+				name: "fluxcd-ns"
+				properties: objects: [{
+					apiVersion: "v1"
+					kind:       "Namespace"
+					metadata: name: "flux-system"
+				}]
+			},
```

Comment:
we need to configure a shared resource for namespace

Reply:
added